velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -336,13 +355,16 @@ VectorPtr VectorFuzzer::fuzzConstant(const TypePtr& type, vector_size_t size) {
         opts_.maxConstantContainerSize.value(), opts_.containerLength);
     opts_.complexElementsMaxSize = std::min<int32_t>(
         opts_.maxConstantContainerSize.value(), opts_.complexElementsMaxSize);
+    // TODO: incorporate fuzzer options into customGenerator.
```

Comment:
This comment feels like it would be more appropriate where we create the InputGeneratorConfig in VectorFuzzer.cpp.

Reply:
Alternatively, JSON really only supports a handful of types: string, integer, float, object (map), array, and boolean (any of which can be null), so JSON could just randomly construct one of those if you don't want to link against VectorFuzzer in the Presto Type code.